management/backup.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python3
+# -*- coding: utf-8 -*-
```

Comment:
It seems strange to me that github is rendering `#!/usr/bin/python3` like it is above.